src/constants/possibleResults.js

```diff
@@ -0,0 +1,37 @@
+export const possibleResults = {
```

Comment:
I can't comment the `SubstanceList.json` that's why this comment is in here haha

The data folder should be outside the assets.